docs/Project-Configuration/Controlling-when-tasks-run.md

```diff
@@ -0,0 +1,63 @@
+# Controlling when tasks run
+
+There are three ways to control the scheduling of builds/tasks on a project's waterfall page.
+
+In short:\
+**Cron:** activates builds/tasks on existing mainline commits based on a specified schedule.\
+**BatchTime:** sets an interval of time in minutes that Evergreen should wait before activating builds/tasks.\
+**Periodic builds:** creates a _new version_ with specified variants/tasks at a specified interval, regardless of commit activity.
+
+Cron and batchtime cannot be combined.
+
+### Cron
```

Comment:
should we link to this where we reference cron in the buildvariant/task section?

Reply:
It has the following on below the two: 
For more on cron and batchtime, see [build variants](../Project-Configuration/Project-Configuration-Files/#build-variants).

Which I think works, because I can't link to just the cron bullet point, and cron and batchtime are mixed together in there.

Reply:
Sorry I meant can we link to _this_ section from that bullet point.